releasenotes/notes/port-elide-permutations-ed91c3d9cef2fec6.yaml

```diff
@@ -0,0 +1,5 @@
+---
+features_transpiler:
+  - |
```

Comment:
I think we haven't actually written release notes like this for any (most of?) the other passes being ported, have we? IMO, we can leave this in for now and the release manager (whoever that ends up being) can just remove this if it's out of place.